kernel/rros/net/ethernet/input.rs

```diff
@@ -20,27 +17,23 @@ fn untag(mut skb: RrosSkBuff, ehdr: &mut bindings::vlan_ethhdr, mac_hdr: *mut u8
             vlan_tci: u16,
         );
     }
-    skb.protocol = ehdr.h_vlan_encapsulated_proto;
+    skb.protocol = ehdr.get_mut().h_vlan_encapsulated_proto;
```

Comment:
Maybe unnecessary mut. We only read fields in ehdr?

Reply:
fix